packages/SwingSet/misc-tools/push-price-times.js

```diff
@@ -0,0 +1,26 @@
+// @ts-nocheck
+import sqlite3 from 'better-sqlite3';
+
+const db = sqlite3('mezzanine.sqlite');
```

Comment:
Assuming we have a need for this script, the SQLite file path should come from argv (and be documented as such).